spec/models/miq_product_feature_spec.rb

```diff
@@ -4,7 +4,7 @@
 
 describe MiqProductFeature do
   before do
-    @expected_feature_count = 872
+    @expected_feature_count = 878
```

Comment:
The rest of the tests validate
* unique identifiers by asserting that we create the feature once and never update other features on a subsequent seeding where nothing else changed in the filesystem
* additional features can be provided via another file in a specific directory